packages/trader/src/Modules/MT5/Containers/mt5-financial-stp-real-account-signup.jsx

```diff
@@ -278,6 +278,7 @@ MT5FinancialStpRealAccountSignup.propTypes = {
 
 export default connect(({ client, modules: { mt5 }, ui }) => ({
     addNotificationByKey: ui.addNotificationMessageByKey,
+    authentication_status: client.authentication_status,
```

Comment:
please add authentication_status propType

Reply:
I'm gonna leave it for now since there's a lot of other propTypes missing and not only `authentication_status` it should be handled when we refactor the component from class to functional .